src/test/java/eu/europeana/metis/sandbox/controller/DatasetControllerIT.java

```diff
@@ -15,34 +9,43 @@
 import org.springframework.mock.web.MockMultipartFile;
 import org.springframework.test.web.servlet.MockMvc;
 
+import java.io.File;
+
+import static eu.europeana.metis.sandbox.common.locale.Country.ITALY;
+import static eu.europeana.metis.sandbox.common.locale.Language.IT;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.multipart;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
 @SpringBootTest(
-    webEnvironment = SpringBootTest.WebEnvironment.MOCK,
-    classes = SandboxApplication.class)
+        webEnvironment = SpringBootTest.WebEnvironment.MOCK,
+        classes = SandboxApplication.class)
 @AutoConfigureMockMvc
 class DatasetControllerIT {
 
-  private final TestUtils testUtils = new TestUtils();
+    private final TestUtils testUtils = new TestUtils();
+
+    @Autowired
+    private MockMvc mvc;
 
```

Comment:
Would be nice to also add scenarios when it is expected to fail or throw an exception.
Also, what about testing the new calls you created?

Reply:
Added new call of harvest with url

Reply:
Done